.github/workflows/release.yml

```diff
@@ -46,3 +79,20 @@ jobs:
           curl -s -H "Authorization: token ${GITHUB_TOKEN}" \
           -d "{\"ref\": \"refs/tags/${{ needs.get-version.outputs.package-version }}\", \"sha\": \"${GITHUB_SHA}\"}" \
           "https://api.github.com/repos/${GITHUB_REPOSITORY}/git/refs"
+
+  update-releases:
+    name: Update Latest release
+    needs:
+      - push-tag
```

Comment:
Is it enough `push-tag` only? I think `build_static_lib` is also needed.

Reply:
Thanks,  it is need.